.github/workflows/update-dapps.yml

```diff
@@ -32,13 +32,14 @@ jobs:
           base: main
           add-paths: |
             src/frontend/src/flows/dappsExplorer/dapps.json
-            src/frontend/assets/icons
+            src/frontend/assets/src/assets/icons
```

Comment:
Ditto, is this correct? I was expecting it to be `frontend/src/assets/icons`

Could you try running this workflow and see that it works as expected?

Reply:
@lmuntaner happy to do that, need to check how to do that though

Reply:
I believe that it's also better to introduce this change along with the PR that introduces caching.

Reply:
Yeah makes sense at the end of the day